src/ui/select-base/selected-items/SelectedItems.scss

```diff
@@ -5,6 +5,7 @@
   height: 54px;
   padding: 0 10px;
   border-top: 1px solid rgba($primary, 0.43);
+  box-shadow: inset -60px 0px 30px -27px #0000007a;
```

Comment:
- rgba() is more convenient to use and to maintain, it also improves legibility
- box-shadow is visible even though only one (and more until it overflows) element is chosen - if it has to indicate that there are more elements available where elements overflow it must be visible on overflow only.
- You might think of some kind of last visible element gradient opacity and hide some part of it in shadow you've done, because now it stays above the shadow which not exactly, same as above, indicates there is something beyond that.

Reply:
2 ok.
- you can add some js code getBoundingReact - and get information on how many width is already available in selected items div. After that compare, this with children count * their current length and apply css class - you can write your own custom hook